source/expectations/data_values/ut_compound_data_value.tpb

```diff
@@ -76,62 +81,54 @@ create or replace type body ut_compound_data_value as
     return l_result_string;
   end;
   
-  member function get_data_diff(a_other ut_data_value, a_exclude_xpath varchar2, a_include_xpath varchar2, 
-                                a_join_by_xpath varchar2, a_unordered boolean) return clob is
-    c_max_rows          constant integer := 20;
+  -- TODO : Rework to exclude xpath 
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this "TODO" comment. ([plsql:S1135](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=801&issues=AWd_NeXxCh2zBQVNm3Cn&open=AWd_NeXxCh2zBQVNm3Cn)

[ISSUE_KEY]: AWd_NeXxCh2zBQVNm3Cn